### ISSUE TITLE
Missing labels on the user deactivation form. 

### ISSUE THREAD

**Issue by Tjoosten** [Activisme-be/Spoon#101]
#### Issue description

There are some missing labels on the user lock form in the application. If we add these we automatically improve the accessibility. 

#### Steps to reproduce the issue

1.  Just check out the input on `resources/view/users/lock.blade.php`
